sig-operations/charter.md

```diff
@@ -0,0 +1,38 @@
+# SIG Operations
+
+## Scope
+
+The goal of this SIG is to enable management and operations for hardware and platform tooling within the Operate First Community Cloud (op1st)
+
+### In scope
+
+* Define what is part and not part of the Operate First Cloud platform offering
+* Communicate a platform roadmap
+* Day to day operations of platform
+* Operating and ensuring stability of:
+    * Platform management tools
+    * Services provided to workloads
+    * Platform component upgrades
+* Coordinate hardware and infrastructure provisioning
+* Resolving platform level incidents
+* Providing GitOps framework for managing platform
+* Providing Automation Runbooks for self servicing repetitive operations tasks
+* PR / Code reviews pertaining to cluster/platform management
+* Documentation of all above
+* Ensure recording decisions via the ADR process
+* Ensure inclusivity and transparency
+
+### Out of scope
+
+* User workload operations
+* Procuring hardware and infrastructure
+
+## Stakeholders
+
+* User workload operations
+* Procuring hardware and infrastructure
+
+### Subproject Creation
+
+Sig Chairs are responsible for creation and removal of subprojects.
```

Comment:
Does this refer to the physical labor in the git repo, or the conceptual creation and removal?

If it's the latter, it seems like this is something that should be done by SIG consensus (both parts) and this reads like the SIG Chair has sole discretion to create or destroy a subproject.

Reply:
++ @quaid . Conceptual creation and removal should be done based on consensus.

Reply:
@quaid since this probably applies to other sigs, can make your suggestion a default in the charter? And then just remove this section here?

Reply:
Well, I figured _someone_ has to be ultimately responsible for coordinating subprojects, bringing creation to a vote, etc. and I figured that responsibility would land on the chairs. But I'll update to sig concensus.

Reply:
Updated, let me know if this is something you'd rather have it as a default, I can remove it from here. (This is something I noticed being mentioned in some sig charters explicitly in the k8s docs).